docs/modules/clients/pages/java.adoc

```diff
@@ -1,5 +1,6 @@
 = Java Client
 :page-api-reference: https://docs.hazelcast.org/docs/{page-latest-supported-java-client}/javadoc
+:page-api-reference: https://docs.hazelcast.org/hazelcast-java-client-docs/{page-latest-supported-java-client-new}/apidocs
```

Comment:
```suggestion

```
I don't think we want to add this. The page-api-reference is only available on 1-1 basis with page so this overwrites the API docs link at top of page from embedded to standalone. We can't have two links here so the choice is none (we take this variable out and rely on hardcoded lnks) or leave it as embedded version for now (until standalone out of beta and/or other page restructuring). WDYT?